### PR TITLE
Rename of gulp task for gateway

### PR DESCRIPTION
**Pull request by yuwzho**
@yaweiw

### CODE REVIEW

File: gateway.js [hunk 1]
```diff
@@ -11,30 +11,30 @@ var all;
...
-  gulp.config = flatten(config);
+  gulp.config = config;
 
   if (typeof all.gulpTaskBI === 'function') {
     all.gulpTaskBI(gulp, 'nodejs', 'gateway', ((options && options.appName) ? options.appName : 'unknown'));
   }
 
   // copy files into profile folder
-  gulp.task('init', 'Initializes sample', function(cb) {
+  gulp.task('init', 'Initialize config files in the user’s profile folder', function(cb) {
```
Comment (yaweiw) [inline gateway.js]: remove the

File: gateway.js [hunk 2]
```diff
@@ -11,30 +11,30 @@ var all;
...
+      saveConfigFile(config.sensortagConfig, options.configTemplate['sensortag-config']);
+      saveConfigFile(config.azureConfig, options.configTemplate['azure-config']);
     }
 
     cb();
   });
 
   // copy inital files to the NUC
-  gulp.task('setup-remote', 'Copy script to remote', function(cb) {
+  gulp.task('install-tools', 'Install necessary tools into the gateway', function(cb) {
```
Comment (yaweiw) [inline gateway.js]: Install necessary tools on the gateway

File: gateway.js [hunk 3]
```diff
@@ -74,7 +74,7 @@ function initTasks(gulp, options) {
   gulp.task('clean', 'clean local and remote', ['clean-remote', 'clean-local']);
 
   // discover sensortag device
-  gulp.task('devdisco', 'discovery Sensortag device', function(cb) {
+  gulp.task('discover-sensortag', 'Discover sensortag devices, need run "install-tools" first', function(cb) {
```
Comment (yaweiw) [inline gateway.js]: Discover TI SensorTag. Run after "install-tools".

File: gateway.js [hunk 4]
```diff
@@ -88,7 +88,7 @@ function initTasks(gulp, options) {
 
   // test sensortag's connectivity
   // usage: gulp testconnect --mac <mac address>
-  gulp.task('testconnect', 'test connectivity of mac address', function(cb) {
+  gulp.task('test-connectivity', 'Test connectivity of the SensorTag, need run "install-tools" first', function(cb) {
```
Comment (yaweiw) [inline gateway.js]: Test connectivity of the SensorTag. Run after "install-tools"

**Comment by yaweiw** [comment]
Approve with comments.

**Review by zikalino — APPROVED** [review]

**Pull request merged**